### PR TITLE
Switched to CircleCI from Travis

### PR DESCRIPTION
**Pull request by serge1peshcoff**
Fixes MEMB-678.
Notable changes: the tests are run not inside the container now, but in the host machine. The container building is still being checked by the `build` step.
Other than that, we should be fine.

(Also I'd rather remove `docker-compose-secrets.yml` and other things, but we can do it later, not a priority for now.)

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+      - checkout
+      - run: docker-compose -f docker/docker-compose-secrets.yml -f docker/docker-compose.yml -f docker/docker-compose.dev.yml build --no-cache oms-core-elixir
+  docker-build-and-push:
+    machine:
+      image: ubuntu-1604:201903-01
+    steps:
+      - checkout
+      - run: docker build  --tag aegee/oms-core-elixir:$(node -p "require('./package.json').version") --tag aegee/oms-core-elixir:latest -f docker/oms-core-elixir/Dockerfile .
+      - run: docker login --username $DOCKER_LOGIN --password $DOCKER_PASSWORD
+      - run: docker push aegee/oms-core-elixir:$(node -p "require('./package.json').version")
```
Comment (linuxbandit) [inline .circleci/config.yml]: isn't there a pipeline env variable that lets you get this metadata? see here as [example of gitlab](https://docs.gitlab.com/ee/ci/variables/predefined_variables.html)

File: .circleci/config.yml [hunk 2]
```diff
@@ -0,0 +1,50 @@
+orbs:
+  slack: circleci/slack@3.4.1
+version: 2.1
+jobs:
+  test:
+    machine:
+      image: ubuntu-1604:201903-01
```
Comment (linuxbandit) [inline .circleci/config.yml]: can we use 18.04?

File: .circleci/config.yml (hunk 1 above)
Comment (serge1peshcoff) [reply .circleci/config.yml]: I was thinking to set the env variable initially, like `export VERSION=$(node -p "require('./package.json')`, but that's tricky and I don't remember why I did this instead. anyway it's the same way in other services.

and afaik, there's no such variable in build-in variables unfortunately

File: .circleci/config.yml (hunk 2 above)
Comment (serge1peshcoff) [reply .circleci/config.yml]: I suppose not, it's not in the list: https://circleci.com/docs/2.0/configuration-reference/#available-machine-images

**Comment by serge1peshcoff** [comment]
will merge tonight, deploy is not required

File: .circleci/config.yml (hunk 1 above)
Comment (linuxbandit) [reply .circleci/config.yml]: https://circleci.com/docs/2.0/env-vars/#built-in-environment-variables

File: .circleci/config.yml (hunk 2 above)
Comment (linuxbandit) [reply .circleci/config.yml]: damn, i thought only travis was using such obsolete thing

**Pull request merged**